tests/php/test-fieldmanager-colorpicker-field.php

```diff
@@ -27,6 +27,32 @@ public function test_basic_render() {
 		$this->assertRegExp( '#<input class="[^"]*fm-colorpicker-popup[^>]+name="test_colorpicker"#', $html );
 	}
 
+	public function test_default_color() {
+		$fm_1 = new Fieldmanager_Colorpicker( 'test-1', array( 'name' => 'test_colorpicker', 'default_color' => '#ff0000' ) );
+		ob_start();
+		$fm_1->add_meta_box( 'Test Colorpicker', 'post' )->render_meta_box( $this->post, array() );
+		$html = ob_get_clean();
+		$this->assertRegExp( '/data-default-color="#ff0000"/', $html );
+
+		$fm_2 = new Fieldmanager_Colorpicker( 'test-1', array( 'name' => 'test_colorpicker', 'default_value' => '#ff0000' ) );
+		ob_start();
+		$fm_2->add_meta_box( 'Test Colorpicker', 'post' )->render_meta_box( $this->post, array() );
+		$html = ob_get_clean();
+		$this->assertRegExp( '/data-default-color="#ff0000"/', $html );
```

Comment:
May as well also check value while you're at it...

```php
$this->assertRegExp( '/data-default-color="#ff0000" value="#ff0000"/', $html );
```

I guess there's no reason not to add this to the others, too